reference/artifacts/types/docker-image/index.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: single
+title:  "Docker Image"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+Docker image artifacts are stored in
+[registries](https://docs.docker.com/registry/), for example
+[GCR](https://gcr.io), or [Docker Hub](https://index.docker.io). They can be
+deployed to Kubernetes or App Engine, and generally trigger pipelines from
+notifications sent by their registry.
+
+## Fields
+
+| Field | Explanation |
+|-|-----------|
+| `type` | Always `docker/image`. |
+| `reference` | The fully-qualified image name, including version and registry. `gcr.io/project/my-image:v1.2` is an example. This can be done either by tag or image digest. |
+| `name` | The image's name (typically the registry and repository) without the image's version. `gcr.io/project/my-image` is an example. |
+| `version` | The image's tag or digest. |
+| `location` | N/A |
```

Comment:
Just wondering&mdash;I'm sure there's a good reason, but why do we need all three fields: reference, name, and version? Is it because if we use the digest for "reference," we can't easily see the name or version?

Reply:
You can't always derive reference from the other fields, and even in cases where you can, there is no artifact-agnostic way to derive version and name from reference.